.gitignore

```diff
@@ -164,3 +164,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# Static Files
+static/
```

Comment:
Same deal here: I think we should add `theme/static/` here, too. Tailwind places its output there and we should probs keep that out of the repo.